src/transforms.py

```diff
@@ -4,30 +4,50 @@
 def get_transforms_list(type: str = "cartesian"):
     if type is "default":
```

Comment:
A minor question: since the default parameter value `"cartesian"` is not implemented otherwise, would it be better to change this `if` statement to `if type is "cartesian"`?

Reply:
good catch!